modules/dream2nix/rust-crane/default.nix

```diff
@@ -207,6 +206,12 @@ in {
   package-func.args = l.mkMerge [common buildArgs];
 
   public = {
+    devShell = import ./devshell.nix {
+      name = "${pname}-devshell";
+      drvs = [cfg.depsDrv.public config.public];
```

Comment:
Ich think it would be better to pass `mainDrv` and `depsDrv` as separate arguments instead of `drvs`;

Reply:
done